scaling_performance/optimizing_compute_resources.adoc

```diff
@@ -107,5 +107,39 @@ Use the *rhel-tools* container with the following:
 
 See the link:https://access.redhat.com/documentation/en/red-hat-enterprise-linux-atomic-host/7/getting-started-with-containers/chapter-11-using-the-atomic-tools-container-image[RHEL Tools Container documentation] for more information.
 
+[[scaling-performance-debugging-using-oa-image]]
+== Debugging {product-title} Using the OpenShift-Ansible Image
+
+Red Hat distributes an https://github.com/openshift/openshift-ansible/blob/master/README_CONTAINER_IMAGE.md[openshift-ansible image], with specific checks focused on detecting common deployment issues.
+Use the following checks to help detect common scaling performance problems:
+
+[[diagnostic-checks]]
+.Diagnostic Checks
+[options="header"]
+|===
+
+|Check Name |Purpose
+
+|`*etcd_imagedata_size*`
+|This check measures the total size of OpenShift image data in an etcd cluster. Fails if the calculated size exceeds a user-defined limit. If no limit is specified, this check will fail if the size of OpenShift image data amounts to `50%`
```

Comment:
I'd avoid saying `50%`, because if we ever decide on tweaking the default value we'd need to remember to update the docs.